R/attr.R

```diff
@@ -110,11 +110,16 @@ detect_void_name <- function(x) {
 is_dictionaryish <- function(x) {
   # 2022-01: Used in many packages. Don't deprecate without a
   # replacement.
-  if (!length(x)) {
-    return(!is.null(x))
-  }
+  if (is.null(x)) {
+    TRUE
```

Comment:
I'd do this with an early return so that the rest of the function is not nested in a branch. So something like:

```r
if (is.null(x)) {
  return(TRUE)
}
if (!length(x)) {
  return(!is.null(x))
}
```

Reply:
done!